test/gtest/conv_trans.cpp

```diff
@@ -43,7 +43,7 @@ void GetArgs(const std::string& param, std::vector<std::string>& tokens)
         tokens.push_back(*begin++);
 }
 
-class ConfigWithFloat : public testing::TestWithParam<std::vector<std::string>>
+class ConfigWithFloat_conv_trans : public testing::TestWithParam<std::vector<std::string>>
```

Comment:
Why we need `ConfigWithFloat_conv_trans`? We are within the `conv_trans` namespace already. Ditto other similar places.

Reply:
@xinlipn What about

> Ditto other similar places.

e.g. `ConfigWithFloat_deepbench_lstm`, `ConfigWithFloat_lstm_extra`?

Reply:
@atamazov , thanks for picking them up. ConfigWithFloat would pass locally, but I recall CI complained about duplicated names, thus the change. 

First reverted back to ConfigWithFloat  in conv_trans.cpp for test. Now reverted ConfigWithFloat  in all files.